src/ziggurat/messaging/producer.clj

```diff
@@ -103,9 +103,20 @@
   "Get exponential backoff enabled for specific channel from config."
```

Comment:
Have you tested this out in integration or production? I'd like to look at the exchanges and queues created in rabbitmq after these changes.

Reply:
This PR only implements exponential backoff for channels, not for normal retry flow. This is an incomplete implementation then.

Reply:
This PR only implements exponential backoff for channels, not for normal retry flow. This is an incomplete implementation then.

Reply:
updated with support for normal flow

Reply:
updated with support for normal flow